
<TranslatedText
	xmlns:cws="http://schemas.cordys.com/cws/1.0"
	type="com.cordys.translation.TranslatedText"
	RuntimeDocumentID="8C1645B9-0967-A1E9-B7D8-8E10DA44E65F">
	<DocumentID>8C1645B9-0967-A1E9-B7D8-8E10DA44A65F</DocumentID>
	<Name>8C1645B9-0967-A1E9-B7D8-8E10DA44A65F</Name>
	<Description>Catch Exception</Description>
	<CreatedBy>sysadmin</CreatedBy>
	<CreationDate>2019-09-25T07:01:26.17</CreationDate>
	<Notes/>
	<Text>Catch Exception</Text>
	<Language>
		<uri
			id="8C1645B9-0967-A1E9-B06A-CC7BA1242657"
		/>
	</Language>
	<TextIdentifier>
		<uri
			id="8C1645B9-0967-A1E9-B7D8-8E10DA44265F"
		/>
	</TextIdentifier>
</TranslatedText>
